tools/cli/src/opt/kanidm.rs

```diff
@@ -212,7 +212,7 @@ pub enum GroupAccountPolicyOpt {
     #[clap(name = "webauthn-attestation-ca-list")]
     WebauthnAttestationCaList {
         name: String,
-        attestation_ca_list_json: String,
+        attestation_ca_list_json_file: PathBuf,
```

Comment:
Yeah, nah this is a backwards breaking change. We try to keep backwards breaking changes to a minimum so ideally we keep allowing the old json string while also allowing the user to provide a file.

```suggestion
        attestation_ca_list_json: Option<String>,
        attestation_ca_list_json_file: Option<PathBuf>,
```

I think the above change would allow the user to keep providing the json through the cli. Might require some clap macro flags to make it work. And will probably need to check that at least one of the arguments was provided in the logic that handles this command.

Reply:
Did not realise the cli wasn't considered stable. Suppose @Firstyear would need to chime in on what he prefers

Reply:
The CLI is not a stable interface, and this change improves usability, so I think I'm happy with it.